### PR TITLE
core: add rados commands options

### LINKED ISSUE #268: `rados` command should be available in the plugin
The plugin should have commands for all ceph tools. Currently the plugin cannot execute `radosgw-admin` as described in [this rook issue](https://github.com/rook/rook/issues/13263#issuecomment-1912677820). Similarly, `rados` should also be exposed. Recently #237 implemented `rados` internally to run commands as part of other commands, but it isn't yet exposed for users to run it as a plugin command.

### PR DESCRIPTION
**Pull request by OdedViner**
**Description of your changes:**
Add rados command option to kubectl-rook-ceph [Krew](https://github.com/kubernetes-sigs/krew) plugin.

**Which issue is resolved by this Pull Request:**
Resolves #268 

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] CI tests has been updated, if necessary.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.

Procedure:
1.Create minikube on my local machine:
minikube start --disk-size=10g --extra-disks=1 --driver kvm2

2.Create Opertaor via conf files from rook repository https://github.com/rook/rook
$ cd deploy/examples/
$ kubectl create -f crds.yaml -f common.yaml -f operator.yaml
$ kubectl get pods -n rook-ceph
NAME                                 READY   STATUS    RESTARTS   AGE
rook-ceph-operator-757cdc49bb-qqffn   1/1     Running   0          89s

3.Create cluster-test:
$ cd deploy/examples/
$ kubectl create -f cluster-test.yaml 
configmap/rook-config-override created
cephcluster.ceph.rook.io/my-cluster created
cephblockpool.ceph.rook.io/builtin-mgr created

$ kubectl get cephcluster -A
NAMESPACE   NAME         DATADIRHOSTPATH   MONCOUNT   AGE   PHASE   MESSAGE                        HEALTH      EXTERNAL   FSID
rook-ceph   my-cluster   /var/lib/rook     1          54m   Ready   Cluster created successfully   HEALTH_OK              abdff5ae-c7cd-41dd-8cd1-661c1db844f8

4.Debug the code with visual studio /.vscode/launch.json
``` 
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "go",
            "request": "launch",
            "mode": "debug",
            "program": "${file}",
            "env": {},
            "args": ["rados", "status"]
        }
    ]
}
```
```
Starting: /home/oviner/go/bin/dlv dap --listen=127.0.0.1:37601 --log-dest=3 from /home/oviner/DEV_REPOS/kubectl-rook-ceph/cmd
DAP server listening at: 127.0.0.1:37601
Type 'dlv help' for list of commands.
Warning: rook version 'rook: v1.14.0-alpha.0.11.g79adfe74f' is running a pre-release version of Rook.

Info: running 'rados' command with args: [df]
POOL_NAME     USED  OBJECTS  CLONES  COPIES  MISSING_ON_PRIMARY  UNFOUND  DEGRADED  RD_OPS     RD  WR_OPS      WR  USED COMPR  UNDER COMPR
.mgr       452 KiB        2       0       2                   0        0         0       6  9 KiB       4  42 KiB         0 B          0 B

total_objects    2
total_used       27 MiB
total_avail      10 GiB
total_space      10 GiB
Process 179702 has exited with status 0
Detaching
```

### CODE REVIEW

**Review by subhamkrai — CHANGES_REQUESTED** [review]
please update commit structure https://github.com/rook/rook/blob/master/Documentation/Contributing/development-flow.md#commit-structure

File: cmd/commands/rados.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
```
Comment (Madhu-1) [inline cmd/commands/rados.go:2]: 2024?

**Comment by subhamkrai** [comment]
> please update commit structure https://github.com/rook/rook/blob/master/Documentation/Contributing/development-flow.md#commit-structure

@OdedViner ^^

**Review by subhamkrai — CHANGES_REQUESTED** [review]
you also need to add docs changes in https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#commands and https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#documentation

**Comment by OdedViner** [comment]
> you also need to add docs changes in https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#commands and https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#documentation

Done

**Comment by subhamkrai** [comment]
> > you also need to add docs changes in https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#commands and https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#documentation
> 
> Done

you missed adding the  https://github.com/rook/kubectl-rook-ceph/blob/master/README.md#documentation

**Review by subhamkrai — CHANGES_REQUESTED** [review]

File: .github/workflows/go-test.yaml [hunk 2]
```diff
@@ -56,6 +56,11 @@ jobs:
         run: |
           set -ex
           kubectl rook-ceph --context=$(kubectl config current-context) ceph status
+      
+      - name: Rados df using context
```
[the remark this answers is not in the document]
Comment (OdedViner) [reply .github/workflows/go-test.yaml:60]: done

**Review by subhamkrai — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**